systems/subscriber/sim-manager/pkg/db/sim_repo.go

```diff
@@ -28,13 +29,15 @@ func NewSimRepo(db sql.Db) SimRepo {
 
 func (s *simRepo) Add(sim *Sim, nestedFunc func(sim *Sim, tx *gorm.DB) error) error {
 	err := s.Db.GetGormDb().Transaction(func(tx *gorm.DB) error {
+		log.Info("Running nested function")
```

Comment:
This line was for debugging purposes. we can remove it.

Reply:
addressed